packages/flame_spine/example/lib/main.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flame/components.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flame_spine/flame_spine.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(GameWidget(game: SpineExampleGame()));
+}
+
+class SpineExampleGame extends FlameGame
+    with HasTappableComponents, DoubleTapDetector {
+  @override
+  Color backgroundColor() {
+    return const Color(0xFFE5E5E5);
+  }
+
+  @override
+  void onDoubleTap() {
+    paused = !paused;
+    super.onDoubleTap();
+  }
+
+  @override
+  Future<void> onLoad() async {
+    final skeleton = await loadSkeleton('spineboy');
```

Comment:
Please add `Spineboy` to `.github/.cspell/flame_dictionary.txt`.

Reply:
or just rename it to spine_boy :P